services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
```

Comment:
The `User` model is imported directly. According to the checklist, you should use the `get_user_model` function for this purpose.